swift/ql/test/library-tests/dataflow/taint/libraries/string.swift

```diff
@@ -462,7 +462,7 @@ func taintFromUInt8Array() {
   let r2 = String(unsafeUninitializedCapacity: 256, initializingUTF8With: {
     (buffer: UnsafeMutableBufferPointer<UInt8>) -> Int in
       sink(arg: buffer[0])
-      sink(arg: taintedUInt8Values[0]) // $ MISSING: tainted=450
+      sink(arg: taintedUInt8Values[0]) // $ tainted=450
```

Comment:
This test was only added in https://github.com/github/codeql/pull/14578 , the case wasn't caught at the time but it looks like it may have benefitted from the closure captures work (https://github.com/github/codeql/pull/14577 or https://github.com/github/codeql/pull/14633).  I believe both branches has CI pass before the other was merged, so neither run included both.